### DISCUSSION TITLE
Question about creating a tunnel with a public server

### DISCUSSION THREAD

**Discussion by JimBeam2019** [alecbcs/hyprspace#75] [Q&A] [answered]
Hi there, can I use hyprspace to create a tunnel between client nodes with a public server? So, the signalling would be done by the server.

**Comment by Winterhuman** [comment, marked as answer]
Sounds like https://github.com/hyprspace/proxy is what you're looking for

**Reply by JimBeam2019** [reply]
> Sounds like https://github.com/hyprspace/proxy is what you're looking for

Thank you for your answer, @LynHyper , and I'd like to ask a follow-up question. What if I try to connect multiple nodes simultaneously through the same public server? Using the same public server, for example, there are nodes A, B, C, D and E, and node A will link to nodes B and C while node D connects to node E. I wondered if it is possible to achieve that with hyprspace and hyprspace proxy.

Cheers.
